cassandra-framework/src/main/resources/logback.xml

```diff
@@ -33,7 +33,7 @@
     <logger name="io.mesosphere" level="trace"/>
 
     <root level="info">
-        <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="STDOUT"/>
```

Comment:
Please remove this so that we won't ship stdout logging when the framework is packaged.

It's there for convenience while developing.